legacy/src/app/factories/spaces.js

```diff
@@ -34,6 +34,9 @@ function SpacesManager(RegionConnection, Manager) {
 
   // Delete the space.
   SpacesManager.prototype.deleteSpace = function (space) {
+    if ("$maasForm" in space) {
+      delete space.$maasForm;
+    }
     return RegionConnection.callMethod("space.delete", space);
```

Comment:
I was going to suggest rewriting this as:
``` ts
return RegionConnection.callMethod("space.delete", { id: space.id });
```
But then I remembered it's legacy and there's a chance that would break something somewhere else.